File: backend/src/main/java/com/wildBirds/BlueChat/api/rest/dto/MessageDto.java
package com.wildBirds.BlueChat.api.rest.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.Instant;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class MessageDto {

        private Long idMessage;
        private String content;
        private Instant sentDate;
        private Long idSender;
        private Long idReceiver;
}

File: mobile/src/app/repository/message/messageDto.ts
import {UserDtoShort} from '../user/userDtoShort';

export class MessageDto {
    private idMessage: number;
    private content: string;
    private sentDate: string;
    private isRead: boolean;
    private sender: UserDtoShort;
    private receiver: UserDtoShort;
}